packages/vendure-plugin-shipping-extensions/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 1.0.0 (2024-01-29)
```

Comment:
Can you copy the changelog and versioning of the previous plugin here? It's basicly the same plugin, but renamed.

Make it a new major version, so that we can use a patch or minor version to deprecate the old plugin on NPM